### ISSUE TITLE
While accessing api url, it shows "Not Found"

### ISSUE THREAD

**Issue by Martin1196155** [Netflix/dispatch-docker#55]
I run the dispatcher with default configuration. Now i am able to access the dispatcher UI. While launch "api url" i am getting {"details":"Not Found"}. I used api url as "http://localhost:80/api/v1".

**Comment by mvilanova** [comment]
@Martin1196155 that's expected. Can you elaborate on what you're trying to do with the API? Also, please report issues with Dispatch in the Dispatch [repo](https://github.com/Netflix/dispatch) in the future.

**Comment by Martin1196155** [comment]
@mvilanova I am trying to configure that api url with the Slack event request URL section. Also for configuring slash commands.

**Comment by mvilanova** [comment]
@Martin1196155 you can find all the information you need to set up the Slack plugin [here](https://hawkins.gitbook.io/dispatch/configuration/plugins/configuring-slack). Let me know if you run into any issues.

**Comment by Martin1196155** [comment]
In the Slack event request URL section I am giving "http://localhost:80/api/v1/events/slack/event". But the http handshake was not happening between dispatch and slack. Because the api returns "not found".

**Comment by mvilanova** [comment]
@Martin1196155 you can't use localhost in the Request URL field. You need to use a publicly accessible, fully qualified domain name (e.g. https://dispatch.example.com/api/v1/events/slack/event).

**Comment by Martin1196155** [comment]
@mvilanova Yes correct, We used ngrok to get the public facing URL and we try to pass that URL in the Slack Request URL section. But i am getting the same issue.

![image](https://user-images.githubusercontent.com/46367679/83220021-814e2980-a18f-11ea-89f0-7c1c3728f9f9.png)

**Comment by mvilanova** [comment]
@Martin1196155 did you set the `SLACK_SIGNING_SECRET` var in the `.env` file? Does the server see the request?

**Comment by Martin1196155** [comment]
@mvilanova Yes we set the SLACK_SIGNING_SECRET var in the .env file. 
![image](https://user-images.githubusercontent.com/46367679/83282828-c9a13200-a1f7-11ea-80cb-7676d9978aef.png)

We get some issues while Running the install.sh script. 

No metric providers specified metrics will not be sent.
ERROR:dispatch.common.utils.cli:Failed to load plugin slack_contact:Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/dispatch/common/utils/cli.py", line 34, in install_plugins
    plugin = ep.load()
  File "/usr/local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2453, in load
    return self.resolve()
  File "/usr/local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2459, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.8/site-packages/dispatch/plugins/dispatch_slack/plugin.py", line 37, in <module>
    from .views import router as slack_event_router
  File "/usr/local/lib/python3.8/site-packages/dispatch/plugins/dispatch_slack/views.py", line 74, in <module>
    slack_client = dispatch_slack_service.create_slack_client()
  File "/usr/local/lib/python3.8/site-packages/dispatch/plugins/dispatch_slack/service.py", line 29, in create_slack_client
    return slack.WebClient(token=SLACK_API_BOT_TOKEN, run_async=run_async)
  File "/usr/local/lib/python3.8/site-packages/slack/web/base_client.py", line 50, in __init__
    self.token = None if token is None else token.strip()
AttributeError: 'Secret' object has no attribute 'strip'

ERROR:dispatch.common.utils.cli:Failed to load plugin slack_conversation:Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/dispatch/common/utils/cli.py", line 34, in install_plugins
    plugin = ep.load()
  File "/usr/local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2453, in load
    return self.resolve()
  File "/usr/local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2459, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.8/site-packages/dispatch/plugins/dispatch_slack/plugin.py", line 37, in <module>
    from .views import router as slack_event_router
  File "/usr/local/lib/python3.8/site-packages/dispatch/plugins/dispatch_slack/views.py", line 74, in <module>
    slack_client = dispatch_slack_service.create_slack_client()
  File "/usr/local/lib/python3.8/site-packages/dispatch/plugins/dispatch_slack/service.py", line 29, in create_slack_client
    return slack.WebClient(token=SLACK_API_BOT_TOKEN, run_async=run_async)
  File "/usr/local/lib/python3.8/site-packages/slack/web/base_client.py", line 50, in __init__
    self.token = None if token is None else token.strip()
AttributeError: 'Secret' object has no attribute 'strip'

WARNING:dispatch.common.utils.cli:Failed to load plugin: zoom_conference Reason: "Config 'ZOOM_API_USER_ID' is missing, and has no default."
Usage: dispatch [OPTIONS] COMMAND [ARGS]...

**Comment by mvilanova** [comment]
@Martin1196155 fixes for these issues are in the `develop` branch of Dispatch. You can point the `docker-compose.yml` file to it until we merge them into master: https://github.com/Netflix/dispatch-docker/blob/master/docker-compose.yml#L20

**Comment by Martin1196155** [comment]
@mvilanova In "docker-compose.yml" file we pointed "develop" branch of Dispatch. Now we are not able to seen the issue which i posted above. However Still i am getting error in slack event request section as below

![image](https://user-images.githubusercontent.com/46367679/83327671-3b7e8780-a29b-11ea-8dbb-a6b2f9199188.png)


In the ngrok we found 403 for every "POST" request.

![image](https://user-images.githubusercontent.com/46367679/83327700-86000400-a29b-11ea-80e2-0d83680a46a5.png)

We double checked the .env file slack plugin configuration. Every thing is fine related to configuration. 

SLACK_APP_USER_SLUG="APP_ID"
SLACK_API_BOT_TOKEN="Bot User OAuth Access Token"
SLACK_SIGNING_SECRET="Signing Secret"
SLACK_WORKSPACE_NAME="Domain name.slack.com"
